### PR TITLE
OFFI-56: Adding Elasticsearch setup workflow

### PR DESCRIPTION
**Pull request by porgabi**
[OFFI-56](https://lombiq.atlassian.net/browse/OFFI-56)

[OFFI-56]: https://lombiq.atlassian.net/browse/OFFI-56?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by wAsnk — APPROVED** [review]

File: test/Lombiq.OSOCE.Tests.UI/Tests/ElasticsearchTests/BehaviorElasticsearchTests.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    {
+    }
+
+    [Fact]
+    public Task ElasticsearchSearchingShouldWork() =>
+        ExecuteTestAfterSetupAsync(
+            async context =>
+            {
+                await context.SignInDirectlyAsync();
+                await context.GoToRelativeUrlAsync("/search");
```
Comment (Piedone) [inline test/Lombiq.OSOCE.Tests.UI/Tests/ElasticsearchTests/BehaviorElasticsearchTests.cs]:
```suggestion
                await context.SignInDirectlyAndGoToRelativeUrlAsync("/search");
```

**Review by wAsnk — APPROVED** [review]

**Comment by wAsnk** [comment]
@porgabi Please fix the windows build, and check Zoltán comment here: https://github.com/Lombiq/GitHub-Actions/pull/376#issuecomment-2248722680

**Comment by Piedone** [comment]
The Windows build still fails: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/10094591094 Please revert until this is fixed, because now dev is broken. The Windows build runs the Elasticsearch test but doesn't enable Elasticsearch. If it's not compatible with Windows, then you can skip it there, see the test filter for security scans.

**Comment by Piedone** [comment]
I've reverted this in dev, because it mustn't have a failing build.

**Pull request merged**